test/run-drun/piggy-bank/ExperimentalCycles.mo

```diff
@@ -1,60 +1,151 @@
-/// Managing cycles
+/// Managing cycles within actors on the Internet Computer (IC).
```

Comment:
There's not much point adding documentation here - it would need to go into motoko-base...

(Maybe just open an issue to do that)

Reply:
Or did you copy this from base?

Reply:
I copied!